package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "google-play-scraper",
-  "version": "6.0.1",
```

Comment:
don't change the version as part of your PR, this should be done only when a version is actually published.